application: serial_lte_modem: Update #XSLMVER command (#10324)
nrfconnect/sdk-nrf: applications/serial_lte_modem/doc/Generic_AT_commands.rst

SLM version #XSLMVER
====================

> The ``#XSLMVER`` command requests the versions of SDK in which the SLM application is built.

Reviewer: ```suggestion
The ``#XSLMVER`` command requests the version of the |NCS| in which the SLM application is built.
```
Should this mention the modem library as well?

Author: Added a line

Set command
-----------

> The set command requests the version of the NCS SDK and that of the modem library.

Reviewer: ```suggestion
The set command requests the version of the |NCS| and that of the modem library.
```
The S in NCS is SDK, so NCS SDK is redundant.

::

   #XSLMVER: <ncs_version>,<libmodem_version>

> The ``<ncs_version>`` value returns a string containing the version of the NCS SDK.

Reviewer: ```suggestion
The ``<ncs_version>`` value returns a string containing the version of the |NCS|.
```
